test/s3-benchmarks/src/main/java/software/amazon/awssdk/s3benchmarks/BaseTransferManagerBenchmark.java

```diff
@@ -171,6 +160,40 @@ private void warmUpDownloadBatch() {
         CompletableFuture.allOf(futures.toArray(new CompletableFuture<?>[0])).join();
     }
 
+    private S3AsyncClient createS3AsyncClient(TransferManagerBenchmarkConfig config) {
+        Long partSizeInMb = config.partSizeInMb() == null ? null : config.partSizeInMb() * MB;
+        Long readBufferSizeInMb = config.readBufferSizeInMb() == null ? null : config.readBufferSizeInMb() * MB;
+        switch (config.s3Client()) {
+            case CRT: {
+                logger.info(() -> "Using CRT S3 Async client");
+                S3CrtAsyncClientBuilder builder = S3CrtAsyncClient.builder()
+                                                                  .targetThroughputInGbps(config.targetThroughput())
+                                                                  .minimumPartSizeInBytes(partSizeInMb)
+                                                                  .initialReadBufferSizeInBytes(readBufferSizeInMb)
+                                                                  .targetThroughputInGbps(config.targetThroughput() == null ?
+                                                                                          Double.valueOf(100.0) :
+                                                                                          config.targetThroughput());
+                if (config.maxConcurrency() != null) {
+                    builder.maxConcurrency(config.maxConcurrency());
+                }
+                return builder.build();
+            }
+            case JAVA: {
+                logger.info(() -> "Using Java-based S3 Async client");
+                return S3AsyncClient.builder()
+                                    .multipartEnabled(true)
+                                    .multipartConfiguration(c -> c.minimumPartSizeInBytes(partSizeInMb * MB)
+                                                                  .thresholdInBytes(partSizeInMb * 2 * MB)
```

Comment:
Is there any reason we are setting threshold here but not crt client?

Reply:
No you are lets use the same for both (default, which is `minimumPartSizeInBytes`)